dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/config/IntegrationTestConfig.java

```diff
@@ -42,16 +40,19 @@
 import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.security.ldap.authentication.LdapAuthenticator;
 import org.springframework.security.ldap.userdetails.LdapAuthoritiesPopulator;
-import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.junit.jupiter.Testcontainers;
 
 /**
  * @author Gintare Vilkelyte <vilkelyte.gintare@gmail.com>
  */
 @Slf4j
 @Configuration
 @ComponentScan( "org.hisp.dhis" )
+@Testcontainers
 public class IntegrationTestConfig
 {
+    public static final String POSTGIS_VERSION = "10-2.5-alpine";
```

Comment:
Not for this PR but should we update? This looks like postgres 10

Reply:
I agree, maybe we should even find a way to sync it with our postgres dependencies

Reply:
Do you mean the driver https://github.com/dhis2/dhis2-core/blob/f788463e90e9c1c285e752ec3a255a068a9abd3f/dhis-2/pom.xml#L100 ?

It works with https://jdbc.postgresql.org/download.html 
| supports PostgreSQL 8.2 or newer

Postgis https://github.com/dhis2/dhis2-core/blob/f788463e90e9c1c285e752ec3a255a068a9abd3f/dhis-2/pom.xml#L101 probably needs to align with the Postgis version we use

Reply:
Resolving this since this will not be part of this PR 😄